packages/twenty-emails/src/common-style.ts

```diff
@@ -22,13 +22,18 @@ const grayScale = {
   gray0: '#ffffff',
 };
 
+const fontColors = {
```

Comment:
I guess this could just be colors. Otherwise it feels duplicate to font.colors sub-object